shapeshift/web
# feat: thor streaming swaps quote

## Description

Does what it says on the box: this allows us to get quotes for the price-optimized streaming swaps, which will now be automagically reflected in the UI

## Pull Request Type


## Issue (if applicable)

N/A

## Risk



Low - this really only passes a new queryparam in, and the way THOR quotes work is both the streaming and regular flavor are returned in the quote response, the latter being unaffected by whether or not a streaming swap is requested in the quote, meaning there is effectively no regressions risks on regular quotes

## Testing



- Try a bigly quote e.g ~2000 USD worth of an UTXO to another UTXO
- Ensure the streaming swap value is higher than the non-streaming flavor

### Engineering



- ☝🏽 

### Operations



- ☝🏽 


## Screenshots (if applicable)

<img width="1639" alt="Screenshot 2023-08-18 at 22 03 07" src="https://github.com/shapeshift/web/assets/17035424/c32a7aaa-ac70-40a3-8d0a-1b7b13c864de">
<img width="1626" alt="Screenshot 2023-08-18 at 22 05 56" src="https://github.com/shapeshift/web/assets/17035424/e501ffd4-911a-4c73-b8b6-b1d76d056440">
